### PR TITLE
fix: Application deployment fails if we click too fast

### PR DESCRIPTION
**Pull request by weili520**
Signed-off-by: TheYoungManLi <cjl@kubesphere.io>

### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes:
Fixes ##2684

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: src/pages/apps/containers/AppDeploy/index.jsx [hunk 1]
```diff
@@ -110,6 +110,7 @@ export default class App extends React.Component {
       ''
     )
     const { name } = this.appStore.detail
+    await Promise.all([this.fileStore.fetch({ version_id: selectAppVersion })])
```
Comment (xuliwenwenwen) [inline src/pages/apps/containers/AppDeploy/index.jsx:113]: `await this.fileStore.fetch({ version_id: selectAppVersion })` is better

Comment (weili520) [reply src/pages/apps/containers/AppDeploy/index.jsx]: done

**Pull request merged**